sample-functions/Nmap/Dockerfile

```diff
@@ -1,8 +1,13 @@
 FROM functions/alpine:latest
 
-RUN apk add --no-cache nmap
+## nmap -A baidu.com
+## when run nmap with `-A` argument
+## if not install nmap-scripts will cause error:
+##
+##  NSE: failed to initialize the script engine:
+##  could not locate nse_main.lua
+##
 
-ENV fprocess="xargs nmap"
+RUN apk add --no-cache nmap nmap-scripts
 
-
-CMD ["fwatchdog"]
+ENV fprocess="xargs nmap"	ENV fprocess="xargs nmap"
```

Comment:
It looks like there was some issue that resulted in 2 `ENV` commands being here. 1 of them should be deleted